templates/default/my.cnf.erb

```diff
@@ -15,12 +12,13 @@ bind-address    = <%= node['ndb']['mysql_ip'] %>
 pid-file        = <%= node['ndb']['log_dir'] %>/mysql_<%= @mysql_id %>.pid
 tmpdir          = /tmp
 
+# Explicit defaults for TS needed by 'airflow initdb'
+explicit_defaults_for_timestamp = 1
 
```

Comment:
This file is not overwritten in case of an update.

Reply:
Removed file as it was never templated